orgs.js

```diff
@@ -21,6 +21,8 @@
           "type": "org"},
         {"name": "node-red",
           "type": "org"},
+        {"name": "IBM-containerized-forecasting-workflow",
```

Comment:
this org does not exist: https://github.com/IBM-containerized-forecasting-workflow